go.mod

```diff
@@ -10,19 +10,19 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.8.1
-	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
 	gotest.tools/v3 v3.0.3
-	k8s.io/api v0.20.7
-	k8s.io/apiextensions-apiserver v0.20.7
-	k8s.io/apimachinery v0.20.7
+	k8s.io/api v0.21.4
+	k8s.io/apiextensions-apiserver v0.21.4
+	k8s.io/apimachinery v0.21.4
 	k8s.io/cli-runtime v0.20.7
-	k8s.io/client-go v0.20.7
-	k8s.io/code-generator v0.20.7
-	knative.dev/eventing v0.25.1-0.20210819160604-19fcdf4b2aed
+	k8s.io/client-go v0.21.4
```

Comment:
We should be able to remove `go-openapi/spec` replace directive since k8s 0.21 newer `kustomize` finally.